src/renderer/api-decorator.js

```diff
@@ -223,22 +223,22 @@ limitations under the License.
         const { uuid, name, parent, entityType } = entityInfo;
         const winIdentity = { uuid, name };
         const parentFrameName = parent.name || name;
-        const eventMap = new Map();
+        const eventMap = [];
 
-        eventMap.set(`window/initialized/${uuid}-${name}`, winIdentity);
+        eventMap.push([`window/initialized/${uuid}-${name}`, winIdentity]);
 
         // main window
         if (uuid === name) {
-            eventMap.set(`application/initialized/${uuid}`);
+            eventMap.push([`application/initialized/${uuid}`, undefined]);
         }
 
-        eventMap.set(`window/dom-content-loaded/${uuid}-${name}`, winIdentity);
-        eventMap.set(`window/connected/${uuid}-${name}`, winIdentity);
-        eventMap.set(`window/frame-connected/${uuid}-${parentFrameName}`, {
+        eventMap.push([`window/dom-content-loaded/${uuid}-${name}`, winIdentity]);
+        eventMap.push([`window/connected/${uuid}-${name}`, winIdentity]);
+        eventMap.push([`window/frame-connected/${uuid}-${parentFrameName}`, {
             frameName: name,
             entityType
-        });
-        eventMap.set(`frame/connected/${uuid}-${name}`, winIdentity);
+        }]);
+        eventMap.push([`frame/connected/${uuid}-${name}`, winIdentity]);
 
         asyncApiCall('raise-many-events', [...eventMap]);
```

Comment:
https://github.com/HadoukenIO/core/pull/389/files#diff-afd6093bc77cc147dad3d61fcfdec439R243 is diff if this is an array no? have we check where this is consumed? 

````
var a = [{s:1, d:2}]
[...a]
// not the same as 
var m = new Map([['s', 1], ['d',2]])
[...m]
````